x-pack/plugins/threat_intelligence/public/components/empty_page/empty_page.tsx

```diff
@@ -22,8 +22,7 @@ interface EmptyPageProps {
 }
 
 export const EmptyPage: VFC<EmptyPageProps> = ({ integrationsPageLink }) => {
-  const docsLink =
-    'https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html';
+  const { docLinks } = useKibana().services;
```

Comment:
To our discussion yesterday about components vs containers, `EmptyPage` with this addition became a container which is fine, but now looking at the component I'd ask why we have `integrationsPageLink` as a prop but doc link we get from inside the component. what about passing docsLink as a prop the same way and move the logic up either to IndicatorsPage or maybe even better to a new EmptyPage container

Reply:
yup I thought about it too, changing this now! 😄

Reply:
I'm not a fan of having an EmptyPage container and an EmptyPage component though. I feel like for each component if we have a container and a component the plugin codebase is going to grow exponentially and it's going to be complicated to navigate...

For me the distinction between components and containers is more from a perspective of _can this component be reused as is_? If we're not planning on reusing the empty page and pass different links to it, why having 2 components (a container and a component)?

Reply:
for this situation I would be even tempted to move all the logic inside the EmptyPage component (therefore make it a container) and remove everything from the IndicatorsPage...

Anyway we can maybe talk a bit more about it? Also that doesn't have to be done for this PR correct? We could clean up the plugin later?

Reply:
yes, let's discuss. I'm ok with moving the logic inside the EmptyPage component and making it a container as you are right, I don't think it will be reused.